### ISSUE TITLE
Postgres

### ISSUE THREAD

**Issue by bryandease** [FbF/Laravel-Blog#18]
Might need to make a note that some of the queries - since they are raw - don't support Postgres.

**Comment by ziazon** [comment]
Yea, so you get errors like these:

SQLSTATE[42601]: Syntax error: 7 ERROR: syntax error at or near "`" LINE 2: YEAR(`published_date`) AS`year`, ^ (SQL: select YEAR(`published_date`) AS`year`, DATE_FORMAT(`published_date`, "%m") AS`month`, MONTHNAME(`published_date`) AS`monthname`, COUNT(*) AS`count`from "fbf_blog_posts" where "fbf_blog_posts"."status" = APPROVED and "fbf_blog_posts"."published_date" <= 2014-07-05 14:02:02 group by DATE_FORMAT(`published_date`, "%Y%m") order by "published_date" desc)

**Comment by bryandease** [comment]
Exactly.  Because some of the code is using raw queries and not Eloquent or the Query Builder.  So obviously the syntax in mySQL and Postgres won't gel together.

**Comment by ziazon** [comment]
Yea... Hopefully this can be resolved soon because I'll need to create my own blog module if I can't get this one to work with postgres (which obviously, is not ideal heh).

**Comment by project707** [comment]
+1 for getting Postgres support ironed out. I can't seem to find anything that works with Postgres, much less as a package for Laravel (of which this seems to be the only potentially working option).
